Gemfile.lock

```diff
@@ -111,6 +111,7 @@ GEM
     debase-ruby_core_source (0.10.2)
     diff-lcs (1.3)
     docile (1.3.0)
+    dotenv (2.5.0)
```

Comment:
Im not sure why there is a dotenv-rails gem - It must do something for rails else it wouldn't exist.  I think we use it elsewhere ?